BrainPortal/app/models/boutiques_portal_task.rb

```diff
@@ -622,4 +628,21 @@ def file_access_symbol
     @_file_access ||= (self.class.properties[:readonly_input_files].present? || self.tool_config.try(:inputs_readonly) ? :read : :write)
   end
 
+  private
+
+  def boutiques_module_information
+    descriptor = self.descriptor_for_final_task_list
+
+    module_information = []
+
+    integrator_modules = descriptor['custom']['cbrain:integrator_modules']
```

Comment:
This code will  crash for any boutiques descriptor that has no integrator module (or no custom section)